### PR TITLE
Add additional GSoC 2018 ideas

### PR DESCRIPTION
**Pull request by Derky**
Changes and additions to our Google Summer of Code 2018 pages.

### CODE REVIEW

**Comment by Derky** [comment]
Added the "Don't merge" label for now while we add some more ideas before the application deadline of 23 January 2018.

**Comment by Persian-phpBB** [comment]
any idea for "Rich Text Editor Integration" or "Website UX/UI" ?

File: src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig [hunk 1]
```diff
@@ -103,6 +102,40 @@
...
+				Replacing the phpBB Database Abstraction Layer (DBAL) with Doctrine DBAL or a similar library. The tasks includes making it possible to use the library's interface directly, and also wrapping it with phpBB's DBAL interface.
+			</p>
+			<h4 class="gsoc-page">Knowledge Prerequisites</h4>
+			<ul>
+				<li>Knowledge of the DBAL library</li>
+				<li>Understanding phpBB's DBAL interface</li>
+				<li>PHP, OOP, database knowledge</li>
+			</ul>
+			<h4 class="gsoc-page">Available mentors</h4>
+			<p>Máté Bartus</p>
```
Comment (marc1706) [inline src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig]: could add me here as well

File: src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig [hunk 2]
```diff
@@ -103,6 +102,40 @@
...
+				<li>An extensible BBCode parser for CKEditor which works with the built in BBCodes and provides an interface to extend it</li>
+				<li>Refactoring the editor backend</li>
+			</ul>
+			<h4 class="gsoc-page">Knowledge Prerequisites</h4>
+			<ul>
+				<li>ES6, JavaScript, OOP knowledge and familiarity with JS build tools.</li>
+				<li>Familiarity with PHP and the phpBB extension system</li>
+			</ul>
+			<h4 class="gsoc-page">Available mentors</h4>
+			<p>Máté Bartus</p>
```
Comment (marc1706) [inline src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig]: same here, would help with this as well

File: src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig (hunk 1 above)
Comment (Nicofuma) [reply src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig]: me too

File: src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig (hunk 2 above)
Comment (hanakin) [reply src/AppBundle/Resources/views/Development/gsoc_ideas.html.twig]: Backup for me on the front stuff here

**Comment by Derky** [comment]
@Persian-phpBB Rich text editor added now and there's already a project for the Admin UI/UX.

@marc1706 @Nicofuma @hanakin Thanks, added you as mentors.

**Pull request merged**